conf/AutoBalance.conf.dist

```diff
@@ -51,7 +51,8 @@ AutoBalance.InflectionPointRaid10MHeroic=0.5
 
 #
 #     AutoBalance.levelScaling
-#        Check the max level of players in map and scale creature based on it
+#        Check the max level of players in map and scale creature based on it.
+#        This triggers depending on the two options below AutoBalance.levelHigherOffset and AutoBalance.levelLowerOffset
 #        0 = Disabled
 #        1 = Enabled only in dungeons
 #        2 = Always enabled
```

Comment:
Please remove "2" as this option is actually treated as boolean in the code, so "0" = false, not "0" = true.

Reply:
1. maybe it's better to change "1 = Enabled only in dungeons" to "1 = Enabled only in dungeons/raids" because "dungeon" in core doesn't mean the 5-player instance, but it's the entire set of instances (including raids), however, it can cause misconceptions to normal users.

2. the "Always enabled" option has not implemented yet and it's a "complex feature" that requires a PR. So, that option can be removed for now, I agree

Reply:
> because "dungeon" in core doesn't mean the 5-player instance, but it's the entire set of instances (including raids), however, it can cause misconceptions to normal users.

yes, stoabrogga explained it to me already but it is confusing for non developers

gonna edit it